examples/features/select-fowrarder/README.md

```diff
@@ -0,0 +1,71 @@
+# Select forwarder by capability example
+
+The example demonstrates how can be declareded forwarder selection in [network service](https://networkservicemesh.io/docs/concepts/architecture/#network-service).
+
+Important points: 
+ - [client](./client.yaml) requests `my-networkservice` service.
+ - [forwarder](./forwarder.yaml) registers itself with label `my_forwarder_capability: true`
+ - [network service](./service.yaml) declaretes network service `my-networkservice` that contains match for label `my_forwarder_capability: true`
+ - [network service-endpoint](./nse-patch.yaml) registers as an [endpoint](https://networkservicemesh.io/docs/concepts/architecture/#network-service-endpoint for service `my-networkservice`
+
+See at example resources in [kustomization.yaml](./kustomization.yaml)
```

Comment:
Do we need to add `kustomization.yaml` to this directory?

Reply:
yes, note: in this example, we do not have dynamic things. So kustomization.yaml could be static.